lib/modules/hosts/pornbot.js

```diff
@@ -20,24 +18,19 @@ addLibrary('mediaHosts', 'pornbot', {
 				poster: info.poster,
 				sources: [{
 					source: info.mp4Url,
-					type: 'video/mp4',
-					class: 'pbRmp4src'
+					type: 'video/mp4'
 				}]
 			};
 			if (info.webmUrl) {
 				video.sources.unshift({
 					source: info.webmUrl,
-					type: 'video/webm',
-					class: 'pbRwebmsrc'
+					type: 'video/webm'
 				});
 			}
-			const element = template.html(video)[0];
-			const v = element.querySelector('video');
 
-			// set the max width to the width of the entry area
-			v.style.maxWidth = `${$(elem).closest('.entry').width()}px`;
-			window.pornbotObject(element, elem.href, info.poster);
-			return element;
+			return modules['showImages'].videoAdvanced(video, {
```

Comment:
Do you think this module could be converted from `type = 'GENERIC_EXPANDO';` to `type = 'VIDEO';`, because then...